### PR TITLE
Add initial support for Siglis zigfred plus smart switch

### PR DESCRIPTION
**Pull request by Atokulus**

### CODE REVIEW

File: zhaquirks/siglis/zigfred.py [hunk 1]
```diff
@@ -261,3 +262,277 @@ def __init__(self, *args, **kwargs):
...
+            242: {
+                PROFILE_ID: 41440,
+                DEVICE_TYPE: 97,
+                INPUT_CLUSTERS: [],
+                OUTPUT_CLUSTERS: [GreenPowerProxy.cluster_id],
+            },
+        },
+    }
+
+    device_automation_triggers = {
```
Comment (dmulcahey) [inline zhaquirks/siglis/zigfred.py:521]: do these actually work?

Comment (Atokulus) [reply zhaquirks/siglis/zigfred.py:521]: Dear @dmulcahey
Yes they do.

Best regards
Markus

**Comment by Atokulus** [comment]
Dear @dmulcahey 
 How are you?
 
 Do you still need anything to merge this pull request?

Best regards
Markus

**Comment by Atokulus** [comment]
Any update regarding merging?

@dmulcahey @

**Review by dmulcahey — APPROVED** [review]

**Pull request merged**